gptcache/manager/eviction.py

```diff
@@ -33,26 +30,18 @@ def check_evict(self):
         return False
 
     def delete(self):
-        mark_ids = self._scalar_storage.get_ids_by_state(state=-1)
-        mark_ids = [i[0] for i in mark_ids]
-        self._scalar_storage.remove_by_state()
+        mark_ids = self._scalar_storage.get_ids(deleted=True)
+        self._scalar_storage.clear_deleted_data()
         self._vector_base.delete(mark_ids)
 
     def rebuild(self):
```

Comment:
Is the call of this method in subsequent development?

Reply:
Next week, we will implement the eviction process. For now, just keep the code correct.